### ISSUE TITLE
No parallel execution for servicebus triggers for void methods when not using dashboard

### ISSUE THREAD

**Issue by jr01** [Azure/azure-webjobs-sdk#918]
I'm using SDK 1.1.2 and don't use the dashboard (config.DashboardConnectionString = null) and in that case the SDK doesn't execute servicebus void methods in parallel.

#### Repro steps

1. Download my repository: https://github.com/jr01/WebJobsParallelIssue -- (this is a simple adaptation of https://github.com/Azure/azure-webjobs-sdk-samples/tree/master/BasicSamples/ServiceBus)
2. Open solution, wait for packages to download
3. Fill in connectionstrings in App.config
4. F5/Run

#### Expected behavior

All messages should start and finish at roughly the same time.

#### Actual behavior

There are 2 messages queued for queue 1, 2 messages for Q2, 2 for Q3.
Notice that Q1 messages are processed serially. 
Notice that Q2 messages are processed serially and are blocked by Q1 messages.
Notice that Q3 messages are processed in parallel and do not start before Q1 and Q2 messages have been processed.

#### Known workarounds

1. Use async methods and await Task.Run(() => {...}); in the method and don't use void methods. 
2. Use dashboard ==> remove/comment the line with config.DashboardConnectionString = null;

#### Related information 

I did a quick look through the SDK src and the problem might be in https://github.com/Azure/azure-webjobs-sdk/src/Microsoft.Azure.WebJobs.Host/Executors/VoidMethodInvoker.cs

```
public Task InvokeAsync(TReflected instance, object[] arguments)
                {
                    _lambda.Invoke(instance, arguments);
                    return Task.FromResult(0);
                }
```

The _lambda.Invoke is synchronous and blocks the thread. The following code could be a solution to the problem, but I haven't tested this and I'm not an expert in the SDK src and also not in c# async, so I might be totally wrong :)
```
public async Task InvokeAsync(TReflected instance, object[] arguments)
                {
                    await Task.Run(() => {
                        _lambda.Invoke(instance, arguments);
                    });
                }
```

https://github.com/Azure/azure-webjobs-sdk/issues/275 -- at end of thread someone mentiones similar behavior.

**Comment by christopheranderson** [comment]
This is fixed in the 2.0.0 preview bits we have available on NuGet. Closing as fixed.

**Comment by jr01** [comment]
Verified against 2.0.0-beta2 @christopheranderson thanks!
